libraries/AP_HAL_ChibiOS/hwdef/X-MAV-AP-H743v2/defaults.parm

```diff
@@ -0,0 +1,11 @@
+#BATTERY MONITOR ON
+BATT_MONITOR 4
+
+#Serial Port defaults
+SERIAL6_PROTOCOL 42  #MSP OSD
```

Comment:
You can move these into the hwdef by using

define DEFAULT_SERIALx_PROTOCOL yy